.github/workflows/push.yml

```diff
@@ -172,15 +172,25 @@ jobs:
           -total-entries-written 300000 \
           -total-entries-read 10000
 
+  go-version:
```

Comment:
Can you explain a bit more why this is needed?

Reply:
It's a workaround to pass the `$GO_VERSION` environment variable to the called workflow, as this is currently not supported. Please read [this](https://github.com/orgs/community/discussions/26671#discussioncomment-3252793) for reference.

Reply:
Ok, got it :+1:.

Can you please add a bit of a comment for this step with the link to mentioned issue?
Looking at that code itself may be confusing for others in the future.

Reply:
Done, but I added a [link to the official documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations).

Reply:
Hi @marcosanchotene - I don't see the comment anywhere, can you check if the correct commit was pushed?

Reply:
Yes, I made a mistake here. It's there now. Sorry!